RIOT-OS/RIOT
# suit: move CoAP endpoints to the example

### Contribution description

Those endpoints are very specific to the example and not useful for general use.
(We don't want an endpoint where we post the Manifest URL to, if we want to post anything, it'd be the manifest)

Move them to the example's CoAP endpoints to avoid confusing users. 


### Testing procedure

The example does behave just as before.

```
main(): This is RIOT! (Version: 2025.01-devel-207-g83569-suit/coap_endpoints)
RIOT SUIT update example application
Running from slot 0
Image magic_number: 0x544f4952
Image Version: 0x00000000
Image start address: 0x00001100
Header chksum: 0x7f7aaea1

Starting the shell
> suit: received URL: "coap://[2001:db8::1]/fw/suit_update/samr34-xpro/riot.suit.latest.bin"
suit_worker: started.
suit_worker: downloading "coap://[2001:db8::1]/fw/suit_update/samr34-xpro/riot.suit.latest.bin"
suit_worker: got manifest with size 485
suit: verifying manifest signature
suit: validated manifest version
Manifest seq_no: 1732626752, highest available: 0
suit: validated sequence number
Formatted component name: 
Comparing manifest offset 1000 with other slot offset
Comparing manifest offset 20800 with other slot offset
validating vendor ID
Comparing 547d0d74-6d3a-5a92-9662-4881afd9407b to 547d0d74-6d3a-5a92-9662-4881afd9407b from manifest
validating vendor ID: OK
validating class id
Comparing 6421399d-f53c-5d59-8534-b04659463f70 to 6421399d-f53c-5d59-8534-b04659463f70 from manifest
validating class id: OK
Comparing manifest offset 1000 with other slot offset
Comparing manifest offset 20800 with other slot offset
SUIT policy check OK.
Formatted component name: 
riotboot_flashwrite: initializing update to target slot 1
Fetching firmware |█████████████████████████| 100%
Finalizing payload store
Verifying image digest
Starting digest verification against image
Install correct payload
Verified installed payload
Verifying image digest
Starting digest verification against image
Verified installed payload
Image magic_number: 0x544f4952
Image Version: 0x6745c940
Image start address: 0x00020900
Header chksum: 0xca4fd729

suit_worker: update successful
```


### Issues/PRs references
